Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -368,27 +368,29 @@ try {
         if ($settings."$_") { $runAlPipelineParams += @{ "$_" = $true } }
     }
 
-    switch($buildMode){
-        'Clean' {
-            $preprocessorsymbols = $settings.cleanModePreprocessorSymbols
+    if ($buildMode -eq 'Translated') {
+        if ($runAlPipelineParams.Keys -notcontains 'features') {
+            $runAlPipelineParams["features"] = @()
+        }
+        Write-Host "Adding translatiopnfile feature"
+        $runAlPipelineParams["features"] += "translationfile"
+    }
 
-            if (!$preprocessorsymbols) {
-                throw "No cleanModePreprocessorSymbols defined in settings.json for this project. Please add the preprocessor symbols to use when building in clean mode or disable CLEAN mode."
-            }
+    if ($runAlPipelineParams.Keys -notcontains 'preprocessorsymbols') {
+        $runAlPipelineParams["preprocessorsymbols"] = @()
+    }
 
-            if ($runAlPipelineParams.Keys -notcontains 'preprocessorsymbols') {
-                $runAlPipelineParams["preprocessorsymbols"] = @()
-            }
+    # REMOVE AFTER April 1st 2025 --->
+    if ($settings.ContainsKey('cleanModePreprocessorSymbols')) {
+        Write-Host "Adding Preprocessor symbols : $($settings.cleanModePreprocessorSymbols -join ',')"
+        $runAlPipelineParams["preprocessorsymbols"] += $settings.cleanModePreprocessorSymbols
+        Write-Warning -message "cleanModePreprocessorSymbols is deprecated. Use Conditional Settings instead, specifying buildModes and the `preProcessorSymbols` setting."
```

Comment:
Suggestion: 
Maybe we should just link to deprecations.md here? Then we can provide the details on how to migrate to the "new way" in there. E.g.: 
```suggestion
        Write-Warning -message "cleanModePreprocessorSymbols will be deprecated after April 1st 2025. Please see https://github.com/microsoft/AL-Go/blob/main/DEPRECATIONS.md for more information"
```